uefi/CHANGELOG.md

```diff
@@ -11,10 +11,21 @@
 - Added `table::{set_system_table, system_table_boot, system_table_runtime}`.
   This provides an initial API for global tables that do not require passing
   around a reference.
+- `MemoryMap::as_raw` which provides raw access to the memory map. This is for
+  example useful if you create your own Multiboot2 bootloader that embeds the
+  EFI mmap in a Multiboot2 boot information structure.
 
 ## Changed
 - `SystemTable::exit_boot_services` is now `unsafe`. See that method's
   documentation for details of obligations for callers.
+- `BootServices::memory_map` changed its signature from \
```

Comment:
nit: no need for `\`, when rendered the markdown will be word-wrapped anyway.

Reply:
My idea here was to prevent line-breaks within the following code-fragment